src/distributions.jl

```diff
@@ -39,4 +39,3 @@ function as(d::Dists.Product, _data::NamedTuple = NamedTuple())
     as(Vector, as(v[1]), n)
 end
 
-as(m::DistributionMeasures.DistributionMeasure) = as(m.d)
```

Comment:
Could you also explain this change?

Reply:
DistributionMeasures.jl adds methods to allow Distributions to be treated as measures. But this turns out to make things more complex in some weird ways. @oschulz and I had some discussions about this and arrived at the approach of focusing on subtypes of `AbstractMeasure`. If you want to do measure theory on something that's *not* `<:AbstractMeasure`, you can either map it into one or use a wrapper.

I don't remember the details of what was complicated about the traits-like approach. I think Oliver hit some weird corner cases in a recent MeasureBase refactoring.